app/src/androidTest/java/com/alexzh/testapp/ui/home/HomeActivityTest.kt

```diff
@@ -18,77 +18,95 @@ import org.junit.runner.RunWith
 @RunWith(AndroidJUnit4::class)
 class HomeActivityTest {
 
-    @Rule
-    @JvmField
+    /**
+     * The {@link ActivityTestRule} provides functional testing of a single {@link Activity}.
+     * The Espresso framework requires it for launching Activity for testings.
+     *
+     * We can configure activity before running in the following ways:
```

Comment:
`We can` or `we need`? Also I think there needs to be more emphasis on the subject so `before running the test`

Reply:
I prefer to use `we can` because everyone should have a room for thinking about each test case. I want to avoid a list of instructions

Reply:
Is it possible to run such test without the ActivityTestRule?

Reply:
No, but we don't need to configure activity before testing any screen. If we want to write E2E test we have no additional configuration, we can create ActivityTestRule and use it without adding initial `Intent` or mocking data